packages/config/src/bridges/near.ts

```diff
@@ -133,7 +143,7 @@ export const near: Bridge = {
         description: 'Contract storing Near state checkpoints.',
       },
     ],
-    risks: [CONTRACTS.UPGRADE_NO_DELAY_RISK],
+    risks: [],
```

Comment:
If MultiSig can upgrade/change instantaneously subcontracts then the risk applies. As it stands Admin can change (and it did so in September) subcontracts of Prover with e.g. adminSstore()